friend.js

```diff
@@ -132,22 +136,18 @@ export class FriendKey {
 
   // @key.expose #foo
   // Make #foo available externally using the key
-  get expose()
-    return (descriptor, get, set) => {
-      let key = descriptor.key;
-      let string = key.toString();
-      if (typeof key !== "privatename") {
-        throw new TypeError(
-          "@expose may only be used with private class elements");
-      }
-      if (this.#names.has(string)) {
-        throw new TypeError("@expose used on the same name repeatedly");
-      }
-      this.#get = get;
-      this.#set = set;
-      this.#names.set(string, key);
-      return descriptor;
+  expose = descriptor => {
+    let key = descriptor.key;
```

Comment:
is it impossible here for the key to be `null` or `undefined`?

Reply:
The key will always be either a String, a Symbol, or a PrivateName.